packages/elements/src/item/index.ts

```diff
@@ -187,6 +189,16 @@ export class Item extends ControlElement {
     }
   }
 
+  /**
+   * Called after the component is first rendered
+   * @param changedProperties Properties which have changed
+   * @returns {void}
+   */
+  protected firstUpdated (changedProperties: PropertyValues): void {
+    super.firstUpdated(changedProperties);
+    registerOverflowTooltip(this, () => this.textContent, () => this.labelEl ? isElementOverflown(this.labelEl) : false);
```

Comment:
We need to check if the host element already has a tooltip and show that as priority

Reply:
this method does that by default

Reply:
Hi guy, thank for review. I still confused, should I check if the host element already has a tooltip, or shouldn't check because `registerOverflowTooltip` method does that by default.

Reply:
`() => this.textContent` covers only `textContent` case. The item can be populated via the `label` as well. This case is not covered.

Reply:
if the `ef-item` has `title` attribute, it will be taken as priority. @TremayneChrist , did you mean that?

Reply:
> `() => this.textContent` covers only `textContent` case. The item can be populated via the `label` as well. This case is not covered.

Good point @goremikins, I didn't think about it before.
Should I add more renderer conditions like this?
```suggestion
    registerOverflowTooltip(this, () => this.textContent || this.label, () => this.labelEl ? isElementOverflown(this.labelEl) : false);
```

Reply:
As for these labels, I doubt about sub-label text ellipsis, should we consider on this too?, or focus only on label.

Reply:
I think it should support subLabel as well

Reply:
> if the `ef-item` has `title` attribute, it will be taken as priority. @TremayneChrist , did you mean that?

Yes. Thanks for clarification 👍